packages/babel-plugin-emotion/package.json

```diff
@@ -14,15 +14,16 @@
     "clean": "rimraf lib"
   },
   "dependencies": {
-    "babel-generator": "^6.26.0",
-    "babel-macros": "^1.0.2",
+    "@babel/helper-module-imports": "7.0.0-beta.31",
+    "babel-macros": "^1.2.0",
     "babel-plugin-syntax-jsx": "^6.18.0",
```

Comment:
is there a reason this plugin is left here with version 6? mixing babel 7 and babel 6 packages together might not work and is considered highly risky

Reply:
`@babel/plugin-syntax-jsx` has a peerDependency on `@babel/core` whereas `babel-plugin-syntax-jsx` doesn't and we want to maintain babel 6 compatibility for now. I understand it's not ideal.